### ISSUE TITLE
train.py 训练模型的速度可以提升吗

### ISSUE THREAD

**Issue by a00147600** [yeyupiaoling/PPASR#78]
[2022-06-13 17:07:14.305492] 数据增强配置：{'type': 'noise', 'aug_type': 'audio', 'params': {'min_snr_dB': 10, 'max_snr_dB': 50, 'noise_manifest_path': 'dataset/manifest.noise'}, 'prob': 0.5}
[2022-06-13 17:07:14.305492] 数据增强配置：{'type': 'speed', 'aug_type': 'audio', 'params': {'min_speed_rate': 0.9, 'max_speed_rate': 1.1, 'num_rates': 3}, 'prob': 1.0}
[2022-06-13 17:07:14.306931] 数据增强配置：{'type': 'shift', 'aug_type': 'audio', 'params': {'min_shift_ms': -5, 'max_shift_ms': 5}, 'prob': 1.0}
[2022-06-13 17:07:14.307345] 数据增强配置：{'type': 'volume', 'aug_type': 'audio', 'params': {'min_gain_dBFS': -15, 'max_gain_dBFS': 15}, 'prob': 1.0}
[2022-06-13 17:07:14.392491] 数据增强配置：{'type': 'specaug', 'aug_type': 'feature', 'params': {'W': 0, 'F': 10, 'n_freq_masks': 2, 'T': 50, 'n_time_masks': 2, 'p': 1.0, 'adaptive_number_ratio': 0, 'adaptive_size_ratio': 0, 'max_n_time_masks': 20, 'replace_with_zero': True}, 'prob': 1.0}
F:\Anaconda3\envs\ppasr\lib\site-packages\paddle\fluid\reader.py:481: UserWarning: DataLoader with multi-process mode is not supported on MacOs and Windows currently. Please use signle-process mode with num_workers = 0 instead
  "DataLoader with multi-process mode is not supported on MacOs and Windows currently." \
W0613 17:07:15.068353  1164 gpu_context.cc:278] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.6, Runtime API Version: 10.2
W0613 17:07:15.092130  1164 gpu_context.cc:306] device: 0, cuDNN Version: 7.6.
[2022-06-13 17:07:17.677516] 训练数据：132735
[2022-06-13 17:07:20.470332] Train epoch: [1/65], batch: [0/4147], loss: 85.51178, learning rate: 0.00005000, eta: 8 days, 16:08:24
[2022-06-13 17:07:57.060972] Train epoch: [1/65], batch: [100/4147], loss: 21.53686, learning rate: 0.00005000, eta: 1 day, 3:22:49
[2022-06-13 17:08:38.158043] Train epoch: [1/65], batch: [200/4147], loss: 19.60888, learning rate: 0.00005000, eta: 1 day, 2:17:50
[2022-06-13 17:09:14.325362] Train epoch: [1/65], batch: [300/4147], loss: 21.28011, learning rate: 0.00005000, eta: 1 day, 3:02:27
[2022-06-13 17:09:49.613506] Train epoch: [1/65], batch: [400/4147], loss: 24.80223, learning rate: 0.00005000, eta: 1 day, 2:22:27

我的配置参数与源代码train.py一致。我的数据量仅仅129.62小时。不过噪声文件noise 把3个压缩包全部解压了。

**Comment by yeyupiaoling** [comment]
你用Linux系统可以多线程读取数据，会快一些。

**Comment by a00147600** [comment]
> 你用Linux系统可以多线程读取数据，会快一些。

仅仅是读取数据需要改动 别的都不变么？因为我的LINUX服务器安装显卡存在问题 所以不得不用WIN跑
